package.json

```
@@ -33,15 +33,15 @@
   },
   "dependencies": {
     "@craftzdog/react-native-buffer": "^6.0.5",
-    "@did.coop/did-key-ed25519": "^0.0.9",
     "@digitalbazaar/ezcap": "^4.1.0",
     "@digitalcredentials/data-integrity": "^2.5.1-beta.1",
     "@digitalcredentials/did-method-key": "^2.0.3",
     "@digitalcredentials/ed25519-signature-2020": "^6.0.0",
-    "@digitalcredentials/ed25519-verification-key-2020": "^4.0.0",
+    "@digitalcredentials/ed25519-verification-key-2020": "5.0.0-beta.2",
     "@digitalcredentials/eddsa-rdfc-2022-cryptosuite": "^1.2.1-beta.6",
     "@digitalcredentials/issuer-registry-client": "^3.0.0",
     "@digitalcredentials/security-document-loader": "^6.0.0",
+    "@digitalcredentials/ssi": "^3.0.5",
```

```suggestion
    "@digitalcredentials/ssi": "^4.0.0",
```
I just published the new 4.0.0 version, so it should be ready to go for this PR.